plugins/modules/ece_cluster_alias.py

```diff
@@ -12,7 +12,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+DOCUMENTATION='''
 
+Updates Elastic Deployment and adds the indicated Alias
+
+Input example:
+
+      port: "{{ deployment_port }}"
+      host: "{{ deployment_host }}"
+      deployment_name: "{{ deployment_name}}"
+      username: "{{ ece_username }}"
+      password: "{{ ece_password }}"
+      alias_name: "{{ alias_name }}"
+
+'''

```

Comment:
not going to block the release here, but please me the documentation block here align to the standards for Ansible, the formatting in the other modules should help, but the official spec is here - https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block